src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -1134,77 +964,55 @@ public BackupInfo GetBackupInfo(TransactionOperationContext context)
 
         private BackupInfo GetBackupInfoInternal(TransactionOperationContext context)
```

Comment:
This can be removed now and you can use `BackupUtils.GetBackupInfo` directly

Reply:
Done